Rehlate/css/styles.css

```diff
@@ -0,0 +1,757 @@
+    align-items: center;
+    font-size: 1.125rem;
+    font-weight: 600;
+    line-height: 1.641rem;
+    gap: 0.75rem;
+  }
+  .mobile {
+    display: none;
+  }
+}
+body {
+  font-size: 16px;
+  font-family: "Lama Sans" !important;
+  background-image: url("../assets/SVG/slide-img-phone.svg");
+  background-size: contain;
+  background-repeat: no-repeat;
+}
+
+.intro {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  margin: 2.75rem 1.25rem;
+}
+.intro h1 {
+  font-size: 2.375rem;
+  font-weight: 600;
+  line-height: 3.063rem;
+  text-align: center;
+  letter-spacing: -0.02em;
+  padding: 0.781rem;
+}
+.intro p {
+  font-size: 1.3rem;
+  font-weight: 100;
+  line-height: 1.875rem;
+  text-align: center;
+  letter-spacing: -0.02em;
+  padding: 0.781rem;
+  font-family: "Lama Sans-thin";
+}
+.intro button {
+  display: flex;
+  flex-direction: row-reverse;
+  justify-content: center;
+  align-items: center;
+  font-family: "Lama Sans" !important;
+  gap: 0.75rem;
+  background-color: #fec430;
+  border: none;
+  border-radius: 15px;
+  padding: 0.75rem 4.5rem;
+  font-size: 1.06rem;
+  font-weight: 300;
+  line-height: 1.5rem;
+  cursor: pointer;
+  color: black;
+  margin: 1.25rem 0;
+}
+.intro > img {
+  margin-left: -14rem;
+}
+.intro > div {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin-top: -9rem;
+}
+
+@media screen and (min-width: 75rem) {
+  body {
+    background-image: url("../assets/SVG/slide-img-desktop.svg");
+  }
+  .intro {
+    margin: 8.25rem 25%;
+    padding-top: 3.625rem;
+  }
+  .intro h1 {
+    font-size: 3.5rem;
+    font-weight: 600;
+    line-height: 4.5rem;
+    margin: 0;
+    padding: 0;
+  }
+  .intro p {
+    font-size: 1.5rem;
+    font-weight: 500;
+    line-height: 2.5rem;
+  }
+}
+.locations {
+  text-align: right;
+  margin-right: 1.25rem;
+}
+.locations .caraousel {
+  display: flex;
+  flex-direction: row-reverse;
+  justify-content: space-between;
+  align-items: center;
+  gap: 1.25rem;
+  overflow-x: auto;
+  height: -moz-max-content;
+  height: max-content;
+  padding-bottom: 7rem;
+}
+.locations .caraousel .caraousel-item {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+.locations .caraousel .caraousel-item img {
+  width: 14.688rem;
+  max-height: 23.125rem;
+}
+@media screen and (min-width: 75rem) {
+  .locations .caraousel .caraousel-item img {
+    width: 23.75rem;
+    max-height: 37.5rem;
+  }
+}
+.locations .caraousel .caraousel-item .caraousel-card {
+  margin-top: -5rem;
+  background-color: white;
+  padding-bottom: 1rem;
+  padding-top: 1.313rem;
+  padding-right: 0.75rem;
+  border-radius: 16px;
+}
+.locations .caraousel .caraousel-item h5 {
+  font-size: 1rem;
+  font-weight: 900;
+  line-height: 1rem;
+}
+.locations .caraousel .caraousel-item p {
+  font-size: 0.75rem;
+  font-weight: 200;
+  line-height: 0.75rem;
```

Comment:
Check the days and price in the `<p>` tag. It should have a font color of` #797979`. Additionally, increase the space between the country name in the `<h5>` tag and the` <p>` tag below it